### PR TITLE
debug with docker in vscode readme

### PR DESCRIPTION
**Pull request by galrou**

### CODE REVIEW

File: docs/DEBUG_WITH_DOCKER_AND_VSCODE.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Instructions
+1. Watch the following video: https://www.youtube.com/watch?v=b78Tg-YmJZI and do as it says
```
Comment (idosavion) [inline docs/DEBUG_WITH_DOCKER_AND_VSCODE.md:2]: Can you please extract the actual required commands from the video?

File: docs/DEBUG_WITH_DOCKER_AND_VSCODE.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+2. Add this line in the dockerfile: ENV FLASK_DEBUG=0
```
Comment (idosavion) [inline docs/DEBUG_WITH_DOCKER_AND_VSCODE.md:3]: why? is this related to the debugger somehow?

File: docs/DEBUG_WITH_DOCKER_AND_VSCODE.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+3. Change the line:
+```CMD python -m ptvsd --host 0.0.0.0 --port 5678 --wait --multiprocess -m flask run -h 0.0.0 -p 5000```
+to ```CMD python -m ptvsd --host 0.0.0.0 --port 5678 --wait --multiprocess -m flask run -h 0.0.0.0 -p 5000```
+4. Comment the lines ```ENTRYPOINT ["/anyway/docker-entrypoint.sh"]``` and ```CMD FLASK_APP=anyway flask run --host 0.0.0.0 ```
+
+5. in launch.json add the following code:
+
+  ```
```
Comment (idosavion) [inline docs/DEBUG_WITH_DOCKER_AND_VSCODE.md:11]: great use of the `launch.json`, but you don't need to add the file, just add the single configuration to the existing configurations, worth mentioning.

File: docs/DEBUG_WITH_DOCKER_AND_VSCODE.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+            ],
+            "port": 5678,
+            "host": "127.0.0.1"
+        }
+  ```
+ 
+ #### Note: everytime you change something in dockerfile you have to rebuild the docker using the command:
+ `sudo docker-compose build`
+ 
+6. in docker-compose.yaml there is this code:
```
Comment (idosavion) [inline docs/DEBUG_WITH_DOCKER_AND_VSCODE.md:35]: This is true, the snippet below exist in our source code, but how is this helping a person coming to debug on vscode?

**Review by idosavion — COMMENTED** [review]
Added a few comments, thanks for adding these stuff! 🎉 
consider adding the instruction from the video into the actual code in `launch.json`

**Comment by atalyaalon** [comment]
@galrou thank you for the pr - it was very helpful and @idosavion thank you for your comments!
I opened a pr for a doc based on the above instructions : https://github.com/hasadna/anyway/pull/1750
closing this one
